packages/core/src/Account.ts

```diff
@@ -198,6 +203,30 @@ export class Account extends TypedEventEmitter<Events> {
     return storeEngine.updateOrCreate(AUTH_TABLE_NAME, AUTH_COOKIE_KEY, entity);
   }
 
+  public async enrollE2EI(
```

Comment:
The main enrollment function for the webapp

Reply:
suggestion: some JSdoc would be nice for this function (that explains roughly the enrolment process)